Gruntfile.js

```diff
@@ -35,7 +35,7 @@ module.exports = function (grunt) {
 
 		mochaIstanbul : {
 			coverage : {
```

Comment:
Prevents helper scripts from being treated as tests. All test suites will have `_spec.js` in the file name.